security_groups/client.tf

```diff
@@ -16,3 +16,14 @@ resource "aws_security_group_rule" "client_egress" {
   from_port                = each.value.port
   to_port                  = each.value.port
 }
+
+resource "aws_security_group_rule" "client_egress_to_load_balancer" {
+  for_each = local.ipa_ports
+
+  security_group_id = aws_security_group.client.id
+  type              = "egress"
+  protocol          = each.value.proto
+  cidr_blocks       = [for ip in var.load_balancer_ips : format("%s/32", ip)]
```

Comment:
With regards to `load_balancer_ips`, is there a technical reason we're assuming the IPs are IPv4?  I'm not suggesting they shouldn't at this point.  I understand this is a pattern we've been following so far.  I'm trying to keep a picture of where we have assumptions in the system that might change.

Reply:
There is no technical reason.  I know both ALBs and NLBs support running in dualstack mode.

Reply:
Can this conversation be resolved, or are you requesting something be changed?